docs/components/DocumentationHeader.tsx

```diff
@@ -167,7 +167,21 @@ const SECTION_LINK_TEXT = css`
   right: 0;
 `;
 
-function SectionContainer({ spaceBetween = 0, spaceAround = 0, children, style, className }) {
+type SectionContainerProps = {
+  children: JSX.Element[];
+  spaceBetween?: number;
+  spaceAround?: number;
+  style?: React.CSSProperties;
+  className?: string;
+};
+
+function SectionContainer({
```

Comment:
Maybe the syntax

```
const SectionContainer: React.FC<SectionContainerProps> = ({ ... }) => (
   <div ...
        ...
   </div>
);
```

would be nicer in general. With that you shouldn't need to define `className`, `style` and `children` because they are added by the `React.FC` (Functional Component) generic.